core/include/clist.h

```diff
@@ -466,6 +468,38 @@ static inline size_t clist_count(clist_node_t *list)
     return cnt;
 }
 
+/**
+ * @brief   Tells if a list has exactly one element
+ *
+ * @note    Complexity: O(1)
+ *
+ * @param[in]   list    Pointer to the clist
+ *
+ * @return      Cardinality == 1 information
```

Comment:
here and below: please drop the "@return", and rename "@retval" to "@returns".

The former because IMO, return and retval are redundant.
The latter because the rest of the file uses `@returns`, and if we want to fix that, we should do all in one go and until then keep it consistent.

Reply:
count also uses return
shall i fix?

Reply:
hm, or just use `@return`and I open a PR that changes the `@returns` once this is in.

Reply:
about RIOT:
```
RIOT$ rg @return |wc -l
6278
RIOT$ rg @returns |wc -l
448
```
the first includes the second so its about 14:1